### ISSUE TITLE
Wrong SQL Syntax in AddColumn for ClickHouse dialect

### ISSUE THREAD

**Issue by iqDF** [go-gorm/clickhouse#9]
## Description

Issue due to wrong SQL  for `ALTER TABLE ? ADD COLUMN ?` as opposed to what is described in the official [Clickhouse docs](https://clickhouse.tech/docs/en/sql-reference/statements/alter/column/#alter_add-column)

Want Correct SQL:
```
ADD COLUMN [IF NOT EXISTS] name [type] [default_expr] [codec] [AFTER name_after]
```

Current SQL:
It doesn't has `...COLUMN...` before the column name. So it will throw syntax error from the ClickHouse server.
```
ADD [IF NOT EXISTS] name [type] [default_expr] [codec] [AFTER name_after]
```

### Fix Suggestion:

```
func(m Migrator) AddColumn(value interface{}, name string) error
```

**Comment by iqDF** [comment]
Fixing in the MR: #8 Fix AddColumn SQL builder. Also add some support for the `ADD COLUMN` options available from ClickHouse syntax.

**Comment by iqDF** [comment]
Solved in Pull Request: #8 Fixed `AddColumn()` builder
